test/metabase/upload_test.clj

```diff
@@ -247,23 +247,21 @@
                             "2022-01-01,2023-02-28,2022-01-01T00:00:00,2023-02-28T00:00:00"
                             "2022-02-01,2023-02-29,2022-01-01T00:00:00,2023-02-29T00:00:00"]))))))
 
-(deftest unique-table-name-test
-  (mt/test-driver (mt/normal-drivers-with-feature :uploads)
+(deftest ^:parallel unique-table-name-test
+  (mt/test-drivers (mt/normal-drivers-with-feature :uploads)
     (testing "File name is slugified"
       (is (=? #"my_file_name_\d+" (#'upload/unique-table-name driver/*driver* "my file name"))))
     (testing "semicolons are removed"
       (is (nil? (re-find #";" (#'upload/unique-table-name driver/*driver* "some text; -- DROP TABLE.csv")))))))
 
 (deftest load-from-csv-table-name-test
   (testing "Upload a CSV file"
-    (mt/test-driver (mt/normal-drivers-with-feature :uploads)
+    (mt/test-drivers (mt/normal-drivers-with-feature :uploads)
       (mt/with-empty-db
         (let [file       (csv-file-with ["id" "2" "3"])]
           (testing "Can upload two files with the same name"
-            ;; Sleep for a second, because the table name is based on the current second
```

Comment:
sleep only matter if you do upload from API because the table name is determined there.

btw, should we have that test using API tho?